website/views.py

```diff
@@ -214,10 +215,10 @@ def resolve_guid(guid, suffix=None):
         if not referent.deep_url:
             raise HTTPError(http.NOT_FOUND)
         if isinstance(referent, PreprintService):
-            return send_from_directory(
-                os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['preprints']['path'])),
-                'index.html'
-            )
+            if referent.provider._id == 'osf' or not referent.provider.domain:
```

Comment:
I could definitely use some help here, as I haven't found the tests for GUID views